frontend/e2e/utils/nginx/conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
General comment - I think I'd expect a file to named `something.conf` rather than just `conf`